proto/tenancy_config/kaba_config.proto

```diff
@@ -4,7 +4,7 @@ package holms.types.tenancy_config;
 option csharp_namespace = "HOLMS.Types.TenancyConfig";
 option go_package = "tenancyconfig";
 
-message KCRConfig {
+message KabaConfig {
```

Comment:
I'd prefer this be called `KabaEncoderConfig` or something indicating it's for the keycards. Kaba is just a brand which makes this pretty vague (just thinking about the next people who will have to make sense of this).